test/integration/scheduler/taint_test.go

```diff
@@ -786,18 +763,55 @@ func TestTaintBasedEvictions(t *testing.T) {
 				}
 			}
 
+			// Regularly send heartbeat event to APIServer so that the cluster doesn't enter fullyDisruption mode.
+			// TODO(Huang-Wei): use "NodeDisruptionExclusion" feature to simply the below logic when it's beta.
+			heartbeatChan := make(chan struct{})
```

Comment:
we don't really need the heartbeatChan at all, the context should suffice since it is created per test.

Reply:
not really, if we reuse the context, before closing the context (called in `defer cleanTest()`), the nodes have been cleaned up (in `cleanNodes()`), so there would still be a couple of attempts updating the deleted node, then it fails the test.